<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #192a32;
font: 500 18pt &quot;IRANYekan Medium&quot;;
color: #EFB238;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLCDNumber" name="lcd_X">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;IRANSansWeb&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QTextEdit" name="textEdit">
      <property name="font">
       <font>
        <family>IRANYekan Medium</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'IRANYekan Medium'; font-size:18pt; font-weight:500; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:14pt; font-weight:400;&quot;&gt;player1     player2&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:14pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLCDNumber" name="lcd_O">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;IRANSansWeb&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="one_player">
      <property name="text">
       <string>1 Player</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QRadioButton" name="two_player">
      <property name="text">
       <string>2 Player</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="btn_reset">
      <property name="text">
       <string>reset</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btn_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>IRANYekan Medium</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #1f3540;
border-radius: 8px; 
border-bottom : 1px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btn_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>IRANYekan Medium</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #1f3540;
border-radius: 8px; 
border-bottom : 1px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="btn_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>IRANYekan Medium</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #1f3540;
border-radius: 8px; 
border-bottom : 1px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="btn_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>IRANYekan Medium</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #1f3540;
border-radius: 8px; 
border-bottom : 1px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="btn_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>IRANYekan Medium</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #1f3540;
border-radius: 8px; 
border-bottom : 1px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="btn_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>IRANYekan Medium</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #1f3540;
border-radius: 8px; 
border-bottom : 1px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="btn_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>IRANYekan Medium</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #1f3540;
border-radius: 8px; 
border-bottom : 1px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="btn_8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>IRANYekan Medium</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #1f3540;
border-radius: 8px; 
border-bottom : 1px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="btn_9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>IRANYekan Medium</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #1f3540;
border-radius: 8px; 
border-bottom : 1px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="about_btn">
      <property name="text">
       <string>About</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
